modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_field.js

```diff
@@ -135,17 +169,26 @@ AUI.add(
 						return instance;
 					},
 
-					setValue: function(value) {
+					setValue: function(values) {
```

Comment:
I know for text-like fields, we don't usually re-render the fields when calling setValue to prevent losing focus. But, in this case, could we benefit from re-rendering so that we could simply this logic? Quickly looking at the code, seems like this logic would be better suited in the soy file. I might be wrong.